### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by osdevisnot**
update dependencies

### CODE REVIEW

**Comment by lukejacksonn** [comment]
Looks good :+1: did you run it through (your own) prettier or something? All trailing commas seem to  have disappeared :P I like them! Perhaps it is worth adding prettier to this project too 🤔

File: static/404.html [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <script>sessionStorage.redirect = location.href</script>
-    <meta http-equiv="refresh" content="0;URL='/'"></meta>
-  </head>
-  <body>
-    <!--
-    This file will be returned by github-pages servers when the user visits
+
```
Comment (lukejacksonn) [inline static/404.html:3]: Does prettier add all this whitespace? 😕

Comment (osdevisnot) [reply static/404.html:3]: I don't think we should run html with prettier. Let me revise.

Comment (osdevisnot) [reply static/404.html:3]: prettier now ignores HTML files. Let me know if we need additional changes before this can get merged.

**Comment by jorgebucaran** [comment]
@osdevisnot Why isn't this merged?

**Comment by osdevisnot** [comment]
This was open from long, I assumed we are not keen on going forward. cc: @lukejacksonn.

**Pull request closed without merging**